sdk/loadtestservice/Azure.Developer.LoadTesting/tests/TestV2ClientTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Developer.LoadTesting.Models;
+using NUnit.Framework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class TestV2ClientTest
+    {
+        [Test]
+        public async Task PathTestAsync()
+        {
+            // RequiredProperty could not set
+            TestV2 testV2 = new TestV2();
+            testV2.Key = "key";
```

Comment:
If we add "key" to the model after it is constructed, it will be added to the request body payload.  I think this is incorrect - a PATCH operation should not allow you to change the unique id of a resource.

Reply:
I opened up an issue to discuss https://github.com/Azure/autorest.csharp/issues/3640